### PR TITLE
qemu: Allow to configure maximum file migration time

### PR DESCRIPTION
**Pull request by okurz**
Related progress issue: https://progress.opensuse.org/issues/58013

### CODE REVIEW

File: doc/backend_vars.asciidoc [hunk 1]
```diff
@@ -93,6 +93,7 @@ QEMUTHREADS;integer;0;Number of cpu threads used by VM
 QEMUVGA;see qemu -device ?;cirrus;VGA device to use with VM
 QEMU_COMPRESS_QCOW2;boolean;1;compress qcow2 images intended for upload
 QEMU_HUGE_PAGES_PATH;string;undef;Define a path to use huge pages (e.g. /dev/hugepages/)
+QEMU_MAX_MIGRATION_TIME;integer;240;Maximum time a migration to file may take for example for snapshot creation before being forcefully aborted.
```
Comment (Martchus) [inline doc/backend_vars.asciidoc]: Maybe it is worth mentioning that this is measured in seconds.

Comment (okurz) [reply doc/backend_vars.asciidoc]: absolutely. I forgot to mention the unit. Fixed that and will merge.

**Comment by Martchus** [comment]
Looks like the change has been pushed directly on master.

**Comment by okurz** [comment]
yes it was. Should have merged instead. However the change is in.

**Pull request closed without merging**